regulations/static/regulations/js/source/views/comment/comment-review-view.js

```diff
@@ -47,7 +47,8 @@ var CommentReviewView = Backbone.View.extend({
 
   editComment: function(e) {
     var section = $(e.target).closest('li').data('section');
-    var options = {id: section, section: section, mode: 'write'};
+    var label = $(e.target).closest('li').find('.comment-section-label').data('section-label');
```

Comment:
It is a bit data attribute heavy, I was trying to follow convention and then realize that there's a more straightforward route. :P